srcpkgs/efibootmgr/files/kernel.d/efibootmgr.post-install

```diff
@@ -28,18 +28,16 @@ major_version=$(echo $PKGNAME | cut -c 6-)
 # look for previous entry for this major kernel version
 existing_entry=$(efibootmgr | grep "Void Linux with kernel ${major_version}")
 
-# get the boot order
-# this is required because when in the next step the existing entry is removed,
-# it is also removed from the order so it needs to be restored later
-bootorder=$(efibootmgr |grep "BootOrder: " |cut -c 12-)
-
 # if existing, remove it
 if [ "$existing_entry" != "" ]; then
-  /etc/kernel.d/post-remove/50-efibootmgr $PKGNAME
+	/etc/kernel.d/post-remove/50-efibootmgr $PKGNAME
+fi
+
+# if using strict efi suffix, rename vmlinuz
+if [ "x${USE_STRICT_EFI_SUFFIX}" = x1 -a -f /boot/vmlinuz-${VERSION} ]; then
```

Comment:
`-n`. And this file is tracked by xbps, so this won't work anyway.

Reply:
What do you mean? Where is it tracked?